### ISSUE TITLE
Disable autofill for passwords, cards, etc.

### ISSUE THREAD

**Issue by ordotools** [brave/brave-browser#19065]
## Description 

I am using NordPass to manage my passwords, addresses, credit cards, etc. I want to disable all autofilling for Brave so that only NordPass suggests autofill options. I am using Mac OS Big Sur.

## Steps to Reproduce

   1. Install the NordPass extension for Brave, and allow it to manage autofill options.
   2. Disable autofill for Brave in `Settings/Additional Settings/Passwords`, `Settings/Additional Settings/Payment methods`, `Settings/Additional Settings/Addresses and more`, 
   3.  Go to any website with an autofill possibility (e.g. the Github sign-in page) and click an input area.

## Actual result:

Both autofill for NordPass and Brave are present, with Brave's dropdown list overlaying NordPass' dropdown list. _Note that Brave does not ask to save passwords, addresses, etc., which is one thing that does work._

## Expected result:

To have no Brave dropdown list present, only NordPass'.

## Reproduces how often: 

This happens for every input that enables autofill.

## Brave version (brave://version info)

| Name | Version |
| --- | --- |
| Brave | 1.30.86 Chromium: 94.0.4606.61 (Official Build) (x86_64) |
| Revision | 418b78f5838ed0b1c69bb4e51ea0252171854915-refs/branch-heads/4606@{#1204} |
| OS | macOS Version 11.6 (Build 20G165) |

## Miscellaneous Information:

The NordPass Extension does work with the latest Safari.

**Comment by corei8** [comment]
I have gotten NordPass to work with Brave without the issue I described, but only after deleting all the passwords and address previously stored by the browser. 

It shouldn't be this hard, but that is the working solution, currently.

**Comment by bsclifton** [comment]
cc: @rebron @fmarier for thoughts

When you add a password manager to Brave (one that has the appropriate manifest permissions), it'll add a new section to
- brave://settings/passwords
- brave://settings/payments
- brave://settings/addresses

![image](https://github.com/brave/brave-browser/assets/4733304/cd1c50f0-00b1-4e66-8812-9122eb91e767)

This is great - but as @corei8 shares above, it doesn't delete the old entries. The person would need to know to go and clean those up.

One suggestion I can think of is adding some additional UI to brave://settings/autofill
![image](https://github.com/brave/brave-browser/assets/4733304/6fde251c-8378-4acf-9bf6-dcab7c6cadb5)

If a password manager is present, we show the management there (similar to above):
`1Password - Password Manager is controlling Passwords/Payment/Addresses`

And then we could potentially have an alert shown, like `You have entries saved for password/payment/address, even though the extension is managing them. Do you want to clean these up?`

WDYT?

**Comment by Brave-Matt** [comment]
Chiming in here — is it possible to have a toggle here (or in `Passwords --> Settings`) as well that simply disables the in-browser password autofill? Seems like an easy way to work around all this.

**Comment by Brave-Matt** [comment]
Another +1 here: https://community.brave.com/t/how-can-i-disable-the-drop-down-menu-that-suggests-auto-fill-data-for-usernames-and-passwords/534140/2

**Comment by sparkidea25** [comment]
this is a real issue, i have both password manager and brave autofill interfacing with eachother.

**Comment by Neriss** [comment]
an odd setting to not be able to change

**Comment by simplerethink** [comment]
Has any progress been made with this? It's a very annoying issue for people who use a password manager, like 1Password. Oftentimes, I cannot select the 1Password autofill option because Brave's autofill windows sits on top, even though I have Brave's autofill turned off.

**Comment by bsclifton** [comment]
OK folks - upstream closed issue a `Won't fix`
> Status: Won't Fix (Intended Behavior)
> This appears to be a Desktop issue, correct? I see that the overlap isn't great.
>
> But this sounds like it's generally working as intended. Above all else, users must have access to the data they stored. If they stored the same data twice or a newer set of data somewhere else, the piece of data that isn't required should be deleted by the user — and only the user.

We can keep open here - @rebron maybe you can help me prioritize. I put on the Papercuts project board as this is something I experience too and it's annoying for everybody.

**Comment by nrpeyton** [comment]
I believe the way this issue was framed upstream caused some confusion about the true nature of the request. The goal is not to have the browser disable access to data the user has stored. Instead, it’s about providing a clearer and more intuitive way to prevent the browser’s built-in autofill prompts from appearing when the user prefers to rely on an external password manager.

The current toggles and language in brave://password-manager/settings are ambiguous. Many users reasonably assume that turning off autofill behaviors (or related sign-in features) there will also stop the password suggestion dropdowns from appearing. The user expectation is that disabling autofill settings (e.g., _`offer to save password` and `sign in automatically`) means the browser will not display its native suggestion pop-ups, not that stored data will be deleted or that the user will lose access to their credentials.

What’s needed is a straightforward, dedicated setting to hide the browser’s autofill prompts while leaving all stored data intact and accessible if the user needs it later. This allows users who prefer third-party password managers to cleanly rely on them, without Brave’s autofill suggestions interfering.

Autofill prompts are a UI feature, not data itself. Stopping the UI from appearing should not be conflated with deleting stored data.  Most people like to dip their toes in the water before fully transitioning to a standalone password manager, so it’s understandable that users do not want to remove all of their saved credentials just to stop a dropdown from appearing.  
 Instead, the browser should offer a toggle that genuinely disables the autofill dropdown, without removing or altering the user’s underlying credentials.

To summarise:

What is being requested: A clear, user-facing control to disable the browser’s autofill prompts for logins/passwords—just like there are for addresses, credit cards, and other autofill data—without losing stored credentials.
Why this matters: Users who prefer external password managers find the overlapping autofill prompts confusing and frustrating. They want to rely solely on their chosen manager’s interface and suggestions.
Not requested: Deletion or hidden access to stored credentials. The user simply wants to stop the autofill pop-up from appearing, not remove their stored data.
Providing this option would improve user experience, reduce confusion, and let users take full advantage of their chosen password management workflows without forcing them to remove data to suppress an unwanted UI element.

**Comment by cowsaresocute** [comment]
+1

**Comment by gitthangbaby** [comment]
I don't have this problem with Bitwarden, the app itself have settings to correct overlapping. Switching to a safe open source solution is one way to fix this.

I actually use both password managers at the same time. So i add passwords twice each time. Then i can enjoy some lightning fast logins by Brave and have password backup.

Took me a lot of time to understand this popular issue is not my issue. My issue is Brave is asking to update too much and popups never close:
* there's no longer an option to never store the password for the specific page if the external password manager is not the default manager but is in control
* the popups cannot be closed -- by pressing the blue button, or "X" button (raised #43849)

**Comment by pihomeserver** [comment]
@gitthangbaby I have Bitwarden extension connected to self hosted Vaultwarden instance and still have the double suggestion. Not related to "open source solution" or other point. Brave does not have an option to disable suggestion, that's the point

**Comment by bswck** [comment]
+1 on fixing this, it's been annoying me in Brave for a long time.

**Comment by gitthangbaby** [comment]
@pihomeserver it is related.  this overlap issue is reported for all those closed source extensions (ProtonPass, NordPass, 1Password, LastPass) and none of them fixed it. They just suggest to [delete the passwords](https://support.1password.com/disable-browser-password-manager/). Bitwarden fixes this with "Display suggestions when icon is selected" option, and there's no double suggestion, in fact, you can use both conveniently, and that's what I benefit from. Here's some [ideas](https://community.bitwarden.com/t/psa-how-to-easily-autofill-using-the-new-browser-extension-ui/77372/1).

**Comment by cwazuidema** [comment]
+1

**Comment by ma1tech** [comment]
+1

**Comment by doilyuser** [comment]
+1

**Comment by DalveenLeandor** [comment]
+1

**Comment by 1chappie** [comment]
+1

**Comment by sniperosx** [comment]
+1

**Comment by atmovantage** [comment]
+1

**Comment by SiobobKipper** [comment]
+1

**Comment by squareyed** [comment]
+1

**Comment by Brave-Matt** [comment]
+1 https://community.brave.com/t/still-unable-to-disable-brave-password-manager-after-installing-third-party-password-manager/631868
cc @rebron to maybe take another pass at this?

**Comment by wmarler** [comment]
I was the one who posted to the Brave forums that @Brave-Matt referred to above ("Still unable to disable brave password manager ..."). I have the Bitwarden extension, and I compared the behavior across Linux, MacOS, and Windows. In my testing, it was only on MacOS that Bitwarden had an entry in brave://password-manager/settings saying "Bitwarden Password Manager is is controlling this setting," and the "Offer to to save passwords and passkeys is toggled on, but disabled. On Windows and Linux this message was not present, and the "Offer to to save passwords and passkeys" is toggled off. 

So perhaps this is MacOS-specific.

**Comment by Brave-Matt** [comment]
+1
https://community.brave.com/t/stopping-the-prompt-for-a-new-password/634920/2

**Comment by kyro-emerson** [comment]
+1

**Comment by haydonryan** [comment]
I think this is related but I'm using the bitwarden extension - but also having the same thing - when logging into reddit for example, it displays my saved logins (in bitwarden) then overlays common usernames / emails from brave on top.    I have password tracking disabled in brave.

**Comment by ptrrdrck** [comment]
+1

**Comment by TheRealBenForce** [comment]
> I think this is related but I'm using the bitwarden extension - but also having the same thing - when logging into reddit for example, it displays my saved logins (in bitwarden) then overlays common usernames / emails from brave on top. I have password tracking disabled in brave.

+1 here

**Comment by akamienski** [comment]
> I think this is related but I'm using the bitwarden extension - but also having the same thing - when logging into reddit for example, it displays my saved logins (in bitwarden) then overlays common usernames / emails from brave on top. I have password tracking disabled in brave.

I've the same problem, it soo annoying!!

**Comment by ElemonoCopilot** [comment]
@bsclifton Can we get a status update for this? It's been 1,383 days since this issue was opened with little activity from Brave's end to help with this issue

**Comment by bsclifton** [comment]
@ElemonoCopilot the last update you can find here which was 251 days ago:
https://github.com/brave/brave-browser/issues/19065#issuecomment-2536583630

Unfortunately, I don't have any other news to share. The issue is still triaged as a `P4`.

**Comment by jbrva** [comment]
My only trick is to scroll down a bit so the Brave autofill goes away but my preferred password manager stays. Otherwise it's a game of mousing around popups. I'd be happy with a keyboard shortcut for "remove password popup."

**Comment by terekcampbell** [comment]
+1

**Comment by onebod** [comment]
Please put a X or ^v to close or minimise it when it pops up - I don't want to disable it.
Taaaa!

**Comment by avelis26** [comment]
I just started using ProtonPass, and same issue, Brave password manager pops up AND ProtonPass pops up and Brave is always on top. This feature should have been implemented long ago. As it stands right now, Brave doesn't play well with other credential managers. Please respect my right to chose a credential manager :P ... oh and this is ironic because Proton places Brave in the top 2 list of recommended Browsers that respect privacy :P

**Comment by CutSnake01** [comment]
### +1

**Comment by Maximepzv** [comment]
+1

**Comment by robnraz** [comment]
So 4 years and no fix?

**Comment by Lioneltristan** [comment]
+1

**Comment by carlosmaranje** [comment]
Just switched over from Firefox. The built-in password manager makes it impossible to use 1Password effectively. 

<img width="434" height="356" alt="Image" src="https://github.com/user-attachments/assets/19183dab-9db2-4fb5-8c3d-3548b8fa364b" />

**Comment by Maximepzv** [comment]
> Just switched over from Firefox. The built-in password manager makes it impossible to use 1Password effectively.
> 
> <img alt="Image" width="434" height="356" src="https://private-user-images.githubusercontent.com/17086444/513359530-19183dab-9db2-4fb5-8c3d-3548b8fa364b.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjQ3ODQ0MTIsIm5iZiI6MTc2NDc4NDExMiwicGF0aCI6Ii8xNzA4NjQ0NC81MTMzNTk1MzAtMTkxODNkYWItOWRiMi00ZmI1LThjM2QtMzU0OGI4ZmEzNjRiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEyMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMjAzVDE3NDgzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM0MmQwMjQ4ZWQ5NTk4NmFhODA4YmQ3Y2M4MDgxODQ0Njc4NjU5NmVmN2Y4NDFhZGU4MWRhMjRmOWI3OWNjYWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.MrAUkErWeAwP3c3KPPeFBhpQZJnfVCcJsUSjdfaNZIU">

Same issue with Bitwarden

**Comment by klugjo** [comment]
+1

**Comment by escmgc** [comment]
almost 5 years for a trivial feature

**Comment by mrandu** [comment]
this issue is still relevant with 3rd party password managers, same exact issue with bitwarden for example.

**Comment by burrella000-ctrl** [comment]
this is pretty sad to hear they haven't done anything for ~5 years about such a simple issue.. maybe i will just go back to edge..

**Comment by Brave-Matt** [comment]
+1 from a email support request:

> _I have proton to auto fill my login. Why does Brave keep blocking my view of Proton? How do I turn this off._

**Comment by echohead84** [comment]
NordVPN user here. Just went through the process of deleting a wealth of old password data from the built-in PW manager, hopefully nothing I'm going to find myself needing down the road...

**Comment by eyesick2k7** [comment]
+1

just setup vaultwarden so im now using bitwarden as my password manager and this is a very annoying issue, i just ended up backing up my passwords locally and deleting them on brave

surprised this is still an issue almost 5 years later

**Comment by mackan64** [comment]
This needs to be prioritized. Its incredibly annoying having your password field showing up when I have no password in brave and using another password manager. This is main reason for looking for brave alternative.

**Comment by andrecarlucci** [comment]
Another sad user here. Please, add the option to disable Brave password filling.

**Comment by aaronshak** [comment]
+1 same issue with proton pass

**Comment by parkerschoenig** [comment]
Is there any way to prioritize this?

**Comment by NancyFolsom** [comment]
This has been driving me crazy ever since I switched to Brave. It makes my 3rd party password vault very hard to use. I never save passwords in Brave. I never want it to save passwords. I knew I unchecked "offer to save passwords."

 I dug into settings again and noticed the option to sign in automatically was checked. I've unchecked it and I think that's solved the overlay problem.


<img width="1264" height="300" alt="Image" src="https://github.com/user-attachments/assets/752b470d-fc3e-4f58-8a2c-3144df362516" />
